js/ng.js

```diff
@@ -92,20 +92,18 @@ addEventListener('deer-form-rendered', event => {
  */ 
 addEventListener('deer-updated', event => {
     const $elem = event.target
-    //Only care about witness form
+    // Only care about witness form
```

Comment:
Not sure what go you here.  None of this code should be edited.  Remove all of these changes from the PR.